convertINFO.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/mendelics/vcf2df/vcfio"
+
+	goparquet "github.com/fraugster/parquet-go"
+	"github.com/fraugster/parquet-go/parquet"
+	"github.com/fraugster/parquet-go/parquetschema"
+)
+
+// ConvertBETA converts vcf to dataframe parquet file with variantkey + beta
+func ConvertBETA(
+	vcf string,
+	outputFolder string,
+) {
+	missingVcfs := checkIfVcfsExist([]string{vcf})
+	if len(missingVcfs) != 0 {
+		log.Fatalf("missing vcfs: %v", missingVcfs)
+	}
+
+	prsName := filepath.Base(vcf)
+	prsName = strings.Replace(prsName, ".vcf.gz", "", -1)
+
```

Comment:
```suggestion
prsName = strings.Replace(prsName, ".", "_", -1)
```
Adicionando essa linha, o código funciona, e o arquivo .parquet é lido no python, e funciona no paribus também.

Reply:
Acho que o parquet não aceita "." nos nomes das colunas.